ui/partials/main/thPinboardPanel.html

```diff
@@ -63,26 +63,29 @@
 
 <!-- Save UI -->
 <div id="pinboard-controls" class="btn-group-vertical"
-     title="{{!hasPinnedJobs() ? 'No jobs pinned' : ''}}">
+     title="{{!hasPinnedJobs() ? 'No pinned jobs' : ''}}">
   <div class="btn-group">
     <button class="btn btn-default btn-xs save-btn"
-          title="{{ !hasPinnedJobs() ? 'No job selected' : 'Save all classification data' }}"
-          ng-click="save()"
-          ng-disabled="!canSaveClassifications()">save
+            title="{{ saveUITitle('classification') }}"
+            ng-click="save()"
+            ng-disabled="!user.loggedin || !canSaveClassifications()">save
```

Comment:
So trying this removal reminded me why I put it in. In edge cases when you are logged in, and log out, the save menu states are stale and don't reflect you are logged out. It appears you need `user.loggedin` in the markup for Save menus to behave correctly and appear disabled in appropriate conditions. I'd like to leave this in on that basis, unless there's some solution I haven't figured out :)

Easy way to reproduce this is to remove the 4 occurrences of `!user.loggedin` tests here above where `canSaveClassifications()` exsists in the markup, login under vagrant, pin a job, select a non-default classification from the dropdown, then log out. Save and Save classification only menus should appear disabled on logout, but they don't. With the user.loggedin markup check left in, they correctly do.

I do still need to fix that console error on initial page load. I'll see if I can get some help in channel for that this week.